app/views/bidding.py

```diff
@@ -114,6 +114,10 @@ def biddings():
             bids = Bid.query.order_by(Bid.time_updated).all()
             bids.reverse()
 
+    if most_popular:
+        bids = Bid.query.order_by(Bid.popularity).all()
+        bids.reverse()
+
```

Comment:
We already got bids set ... Please pay attention to lines 96-115. Let's discuss this logic...